### PR TITLE
Update reporting guidelines.

### PR DESCRIPTION
**Pull request by Carreau**
Now that Public GHSA opening is available on most repo, be clearer that those can be used directly.

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ of security issues.
 ## Reporting vulnerabilities
 
 If you believe you've found a security vulnerability in a [Jupyter Subproject](https://jupyter.org/governance/list_of_subprojects.html),
-please report it to [security@ipython.org](mailto:security@ipython.org).
+you can either:
+ - Directly open a GitHub Security Advisory (GHSA) in the relevant repository
+ - report it to [security@ipython.org](mailto:security@ipython.org) if opening a GHSA is nto possible, or you are unsure
```
Comment (krassowski) [inline security.md:17]:
```suggestion
 - report it to [security@ipython.org](mailto:security@ipython.org) if opening a GHSA is not possible, or you are unsure
```

**Comment by Carreau** [comment]
Failures seem to be 429 too many requests as we check to many `https://github.com/<user>`, though the error message show a single `/` in `https:/github`... I'll try to add a ignore pattern.

**Comment by krassowski** [comment]
In JupyterLab we have `https://github.com/[^/]+/?$` ignore pattern https://github.com/jupyterlab/jupyterlab/blob/main/.github/workflows/linuxtests.yml#L155

**Comment by krassowski** [comment]
But maybe it should be changed on `maintainer-tools` level after all

**Comment by Carreau** [comment]
meh, I guess we can just force-merge. I'm not sure why it still fails.

**Review by jasongrout — APPROVED** [review]

**Comment by krassowski** [comment]
All green, merging!

**Comment by Carreau** [comment]
Thanks !

**Pull request merged**